__tests__/gateFormatter.test.ts

```diff
@@ -8,8 +8,9 @@ import {
     _controlledGate,
     _groupedOperations,
     _classicalControlled,
-    _gateControls,
+    _zoomButton,
```

Comment:
good choice. I was thinking we might want to add other controls in the future, but this works nicely.